src/inversion_scripts/invert.py

```diff
@@ -226,43 +243,25 @@ def do_inversion(
     # space, so we do not need to add 1
     # xhat = 1 + ratio
     xhat = ratio.copy()
-    xhat[:bc_idx] += 1
+    xhat[:OH_idx] += 1
```

Comment:
I think this would add 1 to every element except for the nth element, but it should be using `bc_idx`. which takes into account the bc elements and the OH element. Maybe a better variable name would be `scale_factor_idx`, which tracks the index of elements using scale factors?

Reply:
An aside, is the OH optimization a scale factor or in concentration space?

Reply:
Your reasoning is correct here. I will also update the variable name for clarity.

OH is a scale factor so I have also updated the code and the few lines below to:

```
    # Update scale factors by 1 to match what GEOS-Chem expects
    # xhat = 1 + ratio
    # Notes:
    #  - If optimizing BCs, the last 4 elements are in concentration space,
    #    so we do not need to add 1
    #  - If optimizing OH, the last element also needs to be updated by 1
    xhat = ratio.copy()
    xhat[:scale_factor_idx] += 1
    if prior_err_oh > 0.0:
        xhat[n_elements] += 1
        print(f"xhat[OH] = {xhat[n_elements]}")
```

Reply:
Great, thanks!

Reply:
Now fixed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/67f8f8a651ba6037f3cb882049755eb82ecb32c3